### ISSUE TITLE
Invalid module name

### ISSUE THREAD

**Issue by dlemmermann** [javafxports/openjdk-jfx#167]
Hi,

I am in the middle of moving my FlexGanttFX framework to OpenJFX 11 and I am getting the error message below whenever I try to startup any of my demos. I have no idea how to interpret this message. Any help much appreciated.

```
Error occurred during initialization of boot layer
java.lang.module.FindException: Unable to derive module descriptor for /Users/lemmi/.m2/repository/org/openjfx/javafx-swing/11-ea+23/javafx-swing-11-ea+23.jar
Caused by: java.lang.IllegalArgumentException: javafx.swing.11.ea.23: Invalid module name: '11' is not a Java identifier
```

**Comment by kevinrushforth** [comment]
I think the fix for issue #161 will address this, but @johanvos can confirm. That fix will be in 11-ea+24.

**Comment by johanvos** [comment]
What version of Java are you using? I think you need at least ea23.

**Comment by dlemmermann** [comment]
I am using ea23.

**Comment by johanvos** [comment]
Access to the repository would be very helpful indeed.

**Comment by dlemmermann** [comment]
I have given you access to the FlexGanttFX repo. I assume you received an email.
The branch is “master-11”. It is up to date. This is a multi-module project. The module "FlexGanttFX" is the parent module. You can compile via "maven install". Ignore the two profiles "server" and "local" for now.

**Comment by dlemmermann** [comment]
We can also do a Skype session if you want me to bootstrap you.

**Comment by dlemmermann** [comment]
When things work, you should be able to just launch MSProjectApp.

**Comment by johanvos** [comment]
I see the issue and indeed, this is caused by the fact that there are modules on the modulePath that have no valid module-info.class or no Automatic-Module-Name in the manifest.
This is indeed fixed by #161

**Comment by dlemmermann** [comment]
Awesome, can't wait for the next release then.
